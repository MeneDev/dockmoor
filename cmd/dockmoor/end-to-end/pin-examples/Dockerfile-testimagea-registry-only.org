FROM menedev/testimagea:registry-only
