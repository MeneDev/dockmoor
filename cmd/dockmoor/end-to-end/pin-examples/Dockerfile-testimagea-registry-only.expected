FROM menedev/testimagea:registry-only@sha256:3d4d88675636f0fdf7899e3d3c6f8d5a9cae768e8b7f38f05505d6a88497e7a1
